android/build.gradle

```diff
@@ -16,11 +16,11 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 33
+    compileSdkVersion 31
```

Comment:
[q] @rawbee why are we changing this?

Reply:
I think we missed this somehow. But trying to make it match from this change: https://github.com/underdog-tech/react-native-pinwheel/pull/111/files

Reply:
haha oops: https://github.com/underdog-tech/react-native-pinwheel/pull/107